File: Beecrowd/1149 Somando Inteiros Consecutivos/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int a, n, soma = 0;

        a = sc.nextInt();

        do {
            n = sc.nextInt();
        } while (n <= 0);

        for (int i = a, cont = 0; cont < n; i++, cont++) {
            soma += i;
        }
        System.out.println(soma);

        sc.close();
    }
}

File: Beecrowd/1149 Somando Inteiros Consecutivos/main.cpp
#include <iostream>
 
using namespace std;

int main(){
  int A, N, i, cont, soma = 0;
  
    cin >> A;
    
    do{
      cin >> N;
    }while(N <= 0);

    for(i = A, cont = 0; cont < N; i++){
        soma += i;
        cont++;
    }
    cout << soma << endl;

  return 0;
}